driver/ppm_events_public.h

```diff
@@ -1541,7 +1541,7 @@ enum ppm_syscall_code {
  * Event information enums
  */
 enum ppm_event_category {
-	EC_UNKNOWN = 0,	/* Unknown event created just to fill the pair ENTER/EXIT */
+	EC_UNKNOWN = 0,	/* Unknown event */
```

Comment:
I am not sure about the duality between unknown and not_generated. I think they're basically the same concept.

Reply:
yeah, I know but let's think that this is a patch, not the final solution, we need to find something better after the release for all these flags... defining a new one allow us to leave the code untouched and have a new flag that is under the bitmask, conceptually the higher flags are related to the general event category, while the lower are relative to the syscall category, using this `EC_UNKNOWN` here we have a halfway solution

Reply:
Ok, we can leave `EC_UNKNOWN` just not to create misunderstandings, we will rename it maybe after the release in a cleanup phase :)